src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -179,18 +185,34 @@ object DockerPlugin extends AutoPlugin {
     Cmd("WORKDIR", dockerBaseDirectory)
 
   /**
-    * @param dockerBaseDirectory, the installation directory
+    * @param dockerVersion
+    * @param dockerBaseDirectory the installation directory
+    * @param daemonUser
+    * @param daemonGroup
     * @return ADD command adding all files inside the installation directory
     */
-  private final def makeAdd(dockerBaseDirectory: String): CmdLike = {
+  private final def makeAdd(dockerVersion: Option[DockerVersion], dockerBaseDirectory: String,
+                            daemonUser: String, daemonGroup: String): Seq[CmdLike] = {
 
     /**
       * This is the file path of the file in the Docker image, and does not depend on the OS where the image
       * is being built. This means that it needs to be the Unix file separator even when the image is built
       * on e.g. Windows systems.
       */
     val files = dockerBaseDirectory.split(UnixSeparatorChar)(1)
-    Cmd("ADD", s"$files /$files")
+
+    dockerVersion match {
+      case Some(DockerVersion(major, minor, _, _)) if major >= 17 && minor >= 9 =>
```

Comment:
The `if` statement looks buggy to me. 
If the server has version `18.0` it will yield false, which is not want we want, right?
It must be something like `if majro >= (17 && minor >= 9) || major > 17`

At this point I'm fine with putting this into a if statement. If we have more of this special case, we should
extract this in somethin like a `DockerSupport` object:

```scala
object DockerSupport {

   def chownFlag(version: DockerVersion): Boolean = ...

}

```